sound/sequences/00_sound_player.s

```diff
@@ -2543,10 +2715,14 @@ layer_jump .layer_ED1
 layer_end
 
 .layer_fn_EE1:
+#ifndef DISABLE_MARIO_PITCH_SHIFTING
 layer_setinstr 21
 layer_note1 37, 0x53, 127
 layer_setinstr 15
 layer_note1 37, 0x4e, 64
+#else
+layer_note1 39, 0x37, 95
```

Comment:
you're missing an instrument here, looks like the snore and the whistle are separated?